tests/integration/test_end_to_end_election.py

```diff
@@ -90,8 +117,7 @@ def test_end_to_end_election(self) -> None:
         self.step_2_encrypt_votes()
         self.step_3_cast_and_spoil()
         self.step_4_decrypt_tally()
-        self.compare_results()
-        self.publish_results()
+        self.step_5_publish_and_verify()
```

Comment:
My expectation is that at some point we will add a 
`5_Publish_and_verify.md` to the repository. It could be argued this PR needs to include that.